frontend/packages/kubevirt-plugin/src/selectors/vm-like/next-run-changes.ts

```diff
@@ -84,7 +84,9 @@ export const isBootOrderChanged = (vm: VMWrapper, vmi: VMIWrapper): boolean => {
     const vmDevices = getDevices(vm.asResource());
     const vmiDevices = getVMIDevices(vmi.asResource());
 
-    return vmDevices.every((bootableDevice, index) => _.isEqual(bootableDevice, vmiDevices[index]));
+    return !vmDevices.every((bootableDevice, index) =>
```

Comment:
why not
 `vmDevices.some((bootableDevice, index) => _.isEqual(bootableDevice, vmiDevices[index])`

Reply:
you mean:
`vmDevices.some((bootableDevice, index) => !_.isEqual(bootableDevice, vmiDevices[index])` ?